source/routes/read/month.js

```diff
@@ -0,0 +1,96 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
+/* *
+read month will send response with form 
+{
+    "user": "dave",
+    "month": "May",
+    "bullets": [
+        {bullet1 json objectt},
+        {bullet2 json object}
+    ],
+    "daillys":[
+        {"date": "...", "daily": "..."}
+    ]
+}
+*/
+module.exports = {
+    '/read/month': {
+        methods: ['get'],
+        fn: function (req, res, next) {
+
+            let tempArr = [];
+
+            //create index for query
+            db.createIndex({
+                index: {
+                    fields: ['user']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7423478)
```suggestion
                    fields: ["user"]
```